lib/experimental.ts

```diff
@@ -92,11 +95,12 @@ async function getStaticAnalysisResult(
   targetImage: string,
   archivePath: string,
   dockerfileAnalysis: DockerFileAnalysis | undefined,
+  imageType?: ImageType | undefined,
```

Comment:
Why undefined here, the `localArchive()` passes in `ImageType` as defined

Reply:
But in `distroless` flow we pass `undefined`: https://github.com/snyk/snyk-docker-plugin/pull/225/files/549edc58ae6eae664023543fdcd9d6cfb0feeb94#diff-a29e3cbf53069cd5ba49e11beb3700efL81

Reply:
Actually, I looked into it and I introduced a bug! So before in distroless flow we were assuming `docker-archive`.  I have fixed that now and set `ImageType.DockerArchive` in `distroless()` function, so we are still assuming if `docker-archive:` or `oci-archive:` the default is docker archive.